locales/en-US/app.ftl

```diff
@@ -102,6 +103,84 @@ ListOfPublishedProfiles--uploaded-profile-information-list =
        *[other] Manage these recordings
     }
 
+## MenuButtons Folder
+
+MenuButtons--index--profile-info-uploaded-label = Uploaded:
+MenuButtons--index--profile-info-uploaded-actions = Delete
+MenuButtons--index--meta-info-subtitle = Profile Information
+
+# Used in the meta info menu button, but the label is changed depending on the variable
+# Variables:
+#   $labelVariable (String) - Label for meta info button based on republish and error status
+MenuButtons--index--meta-info-button =
+    .label = { $labelVariable }
+
+MenuButtons--index--full-view = Full View
+MenuButtons--index--cancel-upload = Cancel Upload
+
+# Used in the upload menu button, but the label is changed depending on the variable
+# Variables:
+#   $labelVariable (String) - Label for the 'Upload' button based on republish and error status
+MenuButtons--index-upload-button =
+    .label = { $labelVariable }
+
+MenuButtons--index--revert = Revert to Original Profile
+MenuButtons--index--docs = Docs
+
+MenuButtons--metaInfo--label = Symbols:
+MenuButtons--metaInfo--profile-symbolicated = Profile is symbolicated
+MenuButtons--metaInfo--profile-not-symbolicated = Profile is not symbolicated
+MenuButtons--metaInfo--resymbolicate-profile = Re-symbolicate profile
+MenuButtons--metaInfo--symbolicate-profile = Symbolicate profile
+MenuButtons--metaInfo--attempting-resymbolicate = Attempting to re-symbolicate profile
+MenuButtons--metaInfo--currently-symbolicating = Currently symbolicating profile
+MenuButtons--metaInfo--recording-started = Recording started:
+MenuButtons--metaInfo--interval = Interval:
+MenuButtons--metaInfo--profile-version = Profile Version:
+MenuButtons--metaInfo--buffer-capacity = Buffer Capacity:
+MenuButtons--metaInfo--buffer-duration = Buffer Duration:
+
+# Buffer Duration in Seconds in Meta Info Panel
+# Variable:
+#   $configurationDuration (Number) - Buffer Duration in Seconds
+MenuButtons--metaInfo--buffer-duration-seconds = { $configurationDuration } seconds
```

Comment:
You want a [plural string](https://www.projectfluent.org/fluent/guide/selectors.html) here, even for English it's incorrect for "1 seconds"